src/Tribe/Assets.php

```diff
@@ -181,7 +181,10 @@ public function filter_modify_to_module( $tag, $handle ) {
 		}
 
 		// Do not add Module to a Script that already has a type.
-		if ( false !== strpos( $tag, ' type=' ) ) {
+		if (
+			false !== strpos( $tag, ' type=' )
+			&& current_theme_supports( 'html5', 'script' ) // These themes already have the `type="text/javascript"` added by WordPress core.
```

Comment:
Should this be `! current_theme_supports()` ?

Reply:
I don't think either of these will solve the issue alone.

At this point we already know that this is a module and that the tag has an attribute, so we need to replace that attribute to `type=module` instead of just returning it.

Reply:
I was just keeping the if (checking if the type is defined), in case it touches some other thing and potentially breaks something.

At this point, we know that is a tribe asset of type module, but WP may have changed the HTML already making this of type="text/javascript" (what was happening, actually).

I'm alright with removing this `if` completely, but I was being cautious of not removing something that could potentially break other stuff.